docs/plugin/troubleshooting.md

```diff
@@ -0,0 +1,89 @@
+
+  ```bash
+  INFO[2023-01-09T21:21:24+01:00] Starting Plugin Manager for all enabled plugins  component="Plugin Manager" enabledSources=botkube/cm-watcher
+  INFO[2023-01-09T21:21:24+01:00] Downloading plugin.                           binPath=/tmp/plugins/botkube/source_v0.13.0_cm-watcher component="Plugin Manager" url="http://host.k3d.internal:3000/static/source_cm-watcher_darwin_amd64"
+  INFO[2023-01-09T21:21:24+01:00] source plugin registered successfully.        binPath=/tmp/plugins/botkube/source_v0.13.0_cm-watcher component="Plugin Manager" plugin=botkube/cm-watcher version=v0.13.0
+  INFO[2023-01-09T21:21:25+01:00] Start source streaming...                     pluginName=botkube/cm-watcher sources="[plugin-based]"
+  ```
+
+  If you don't see any of the above log messages:
+
+  - Make sure that `source.{group_name}.{plugin_name}.enabled` property is set to `true`
+  - Make sure that a given source configuration (`sources.{group_name}`) is bind to a given communication platform (`bindings.sources: [{group_name}]`)
+
+  If the source is not bound to any communication platform Botkube will not download and start such plugin. Even if it's enabled.
+
+- [Make sure that your plugin didn't crash](#plugin-process-exited).
+
+### You don't receive executor response
+
+**Symptoms**
+
+You run a given executor command in a chat window, but you don't get any response.
+
+**Debugging steps**
+
+- [Enable debugging mode for a given plugin](debug.md).
+  Once enabled, run a given executor command once again, and check the Botkube logs. You can filter them by your plugin name. You should see information about downloading and registering your executor. For example:
+
+  ```bash
+  INFO[2023-01-09T21:21:24+01:00] Starting Plugin Manager for all enabled plugins  component="Plugin Manager" enabledExecutors=botkube/echo
+  INFO[2023-01-09T21:21:24+01:00] Downloading plugin.                           binPath=/tmp/plugins/botkube/executor_v0.13.0_echo component="Plugin Manager" url="http://host.k3d.internal:3000/static/executor_echo_darwin_amd64"
+  INFO[2023-01-09T21:21:24+01:00] executor plugin registered successfully.      binPath=/tmp/plugins/botkube/executor_v0.13.0_echo component="Plugin Manager" plugin=botkube/echo version=v0.13.0
+  ```
+
+  If you don't see any of the above log messages:
+
+  - Make sure that `executors.{group_name}.{plugin_name}.enabled` property is set to `true`
+  - Make sure that a given executor configuration (`executors.{group_name}`) is bind to a given communication platform (`bindings.executors: [{group_name}]`)
+
+  If the executor is not bound to any communication platform Botkube will not download and start such plugin. Even if it's enabled.
+
+- [Make sure that your plugin didn't crash](#plugin-process-exited).
+
+### Plugin process exited
+
+**Symptoms**
+
+In Botkube logs, you will see an error similar to:
```

Comment:
I'm not a linguist, but i think this troubleshooting section should be in Present Simple:
```suggestion
In Botkube logs, you see an error similar to:
```

Same in other places (e.g. "Plugin not found error")